client/checkout/stripe-link/index.js

```diff
@@ -44,42 +44,47 @@ const enableStripeLinkPaymentMethod = ( options ) => {
 		const fillWith = options.fill_field_method
 			? options.fill_field_method
 			: ( address, nodeId, key ) => {
-					document.getElementById( nodeId ).value =
-						address.address[ key ];
+					if ( null !== document.getElementById( nodeId ) ) {
+						document.getElementById( nodeId ).value =
+							address.address[ key ];
+					}
 			  };
 
-		if ( options.complete_shipping ) {
+		if ( options.complete_shipping() ) {
 			fillWith( shippingAddress, options.shipping_fields.line1, 'line1' );
 			fillWith( shippingAddress, options.shipping_fields.line2, 'line2' );
 			fillWith( shippingAddress, options.shipping_fields.city, 'city' );
-			fillWith( shippingAddress, options.shipping_fields.state, 'state' );
 			fillWith(
 				shippingAddress,
-				options.shipping_fields.postal_code,
-				'postal_code'
+				options.shipping_fields.country,
+				'country'
 			);
+			jQuery( 'select' ).trigger( 'change' );
+			fillWith( shippingAddress, options.shipping_fields.state, 'state' );
 			fillWith(
 				shippingAddress,
-				options.shipping_fields.country,
-				'country'
+				options.shipping_fields.postal_code,
+				'postal_code'
 			);
 		}
-		if ( options.complete_billing ) {
+		if ( options.complete_billing() ) {
 			fillWith( billingAddress, options.billing_fields.line1, 'line1' );
 			fillWith( billingAddress, options.billing_fields.line2, 'line2' );
 			fillWith( billingAddress, options.billing_fields.city, 'city' );
-			fillWith( billingAddress, options.billing_fields.state, 'state' );
 			fillWith(
 				billingAddress,
-				options.billing_fields.postal_code,
-				'postal_code'
+				options.billing_fields.country,
+				'country'
 			);
+			jQuery( 'select' ).trigger( 'change' );
+			fillWith( billingAddress, options.billing_fields.state, 'state' );
 			fillWith(
 				billingAddress,
-				options.billing_fields.country,
-				'country'
+				options.billing_fields.postal_code,
+				'postal_code'
 			);
 		}
+		jQuery( 'select' ).trigger( 'change' );
```

Comment:
Could we use a class or ID instead of `select`? We shouldn't trigger a change event for all `select` elements on the page.